tsdb/db.go

```diff
@@ -812,6 +812,7 @@ func (db *DB) Compact() (returnErr error) {
 		mint := db.head.MinTime()
 		maxt := rangeForTimestamp(mint, db.head.chunkRange.Load())
 
+		start := time.Now()
```

Comment:
We should count this entire loop and the WAL truncation. So this should be before the loop.